INSTALL.asciidoc

```diff
@@ -45,15 +60,27 @@ cd /tmp/os-autoinst-run
 --------------------------
 7.  create a minimal "vars.json" config file
 +
+.vars.json
+[source,json]
 ---------------------------------------------------------
-cat <<EOF >vars.json
```

Comment:
It is does not work in Fish and the user will need to change it anyway to correct the file paths, which they will probably do in an editor. So I think that it mainly serves as a distraction, but I suppose most OpenQA users will not be bothered by it either way.